modules/darwin/src/main/ts/ephox/darwin/api/SelectionAnnotation.ts

```diff
@@ -55,7 +55,7 @@ const byAttr = (ephemera: Ephemera, onSelection: (cells: SugarElement[], start:
   };
 
   const clearBeforeUpdate = (container: SugarElement) => {
-    const sels = SelectorFilter.descendants(container, ephemera.selectedSelector);
+    const sels = SelectorFilter.descendants(container, `${ephemera.selectedSelector},${ephemera.firstSelectedSelector},${ephemera.lastSelectedSelector}`);
```

Comment:
```suggestion
    const sels = SelectorFilter.descendants(container, [ ephemera.selectedSelector, ephemera.firstSelectedSelector, ephemera.lastSelectedSelector ].join(',')`);
```
I don't have strong feelings either way, but this is an option

Reply:
Thanks for the suggestion. I think I'll stick with the template string since no more selectors should ever be added.